backend/app/api/routes/conversations.py

```
@@ -0,0 +1,487 @@
+    
+    prompt_config = ANALYSIS_PROMPTS[analysis_request.prompt_type]
+    
+    try:
+        # 检查是否已有相同类型的分析对话
+        existing_conversation = session.exec(
+            select(AIConversation).where(
+                AIConversation.content_item_id == content_id,
+                AIConversation.user_id == current_user.id,
+                AIConversation.conversation_type == "prompt_analysis",
+                AIConversation.title == prompt_config["title"],
+                AIConversation.is_active == True,
```

Ruff (0.11.9)

Avoid equality comparisons to `True`; use `if AIConversation.is_active:` for truth checks

Replace with `AIConversation.is_active`

(E712)